app.py

```diff
@@ -115,119 +120,23 @@ def get_frame_range_from_shotgun(self):
 
         return ( data[sg_in_field], data[sg_out_field] )
 
-
     def get_current_frame_range(self, engine):
```

Comment:
While we're in here, let's actually go ahead and add method docstrings here too -- this will let you document the exceptions that could be raised.

```suggestion
     def get_current_frame_range(self, engine):
        """
        Gets the current scene frame range.

        :param engine: The current engine.

        :returns: Returns the frame range in the form (in_frame, out_frame)
        :rtype: tuple[int, int]

        :raises TankError: Raised if an error is encountered when getting the frame range for the current engine.
        """
```

Reply:
Done